### PR TITLE
Added support of room_id option

### PR DESCRIPTION
**Pull request by kirs**
Instead of `room` option, you can pass the `room_id` if your room name contains some special characters.

### CODE REVIEW

**Comment by technicalpickles** [comment]
Sorry for slow response, maintaining open source is hard :sweat: 

This looks like a good addition. Off hand, do you have any examples of said special cases? I'd also like to see some updated docs to specify room_id and specs to confirm it working.

I added support for multiple rooms awhile back, so this isn't going to merge cleanly anymore.

File: lib/capistrano/campfire.rb [hunk 1]
```diff
@@ -13,14 +13,20 @@ def self.extended(configuration)
...
 
           campfire = ::Tinder::Campfire.new account,
             :token => token,
             :ssl => ssl,
             :ssl_verify => ssl_verify
 
-          campfire.find_room_by_name(room_name)
+          if room_name
+            campfire.find_room_by_name(room_name)
+          else
```
Comment (technicalpickles) [inline lib/capistrano/campfire.rb]: Now that you can specify by room_name or room_id, should make this `elsif room_id`, with an else clause that raises an error.

**Comment by kirs** [comment]
@technicalpickles for example room name can be in russian or in chinese and it would be not so clear to find_room_by_name("название моей комнаты")

**Comment by technicalpickles** [comment]
Makes sense. That may be an upstream bug for tinder and/or Campfire too, since that should just work.

**Comment by kirs** [comment]
@technicalpickles fixed.

**Comment by technicalpickles** [comment]
@kirs gah, sorry again :( I added you as a collaborator if you'd like to merge. If you want to send the email on your rubygems account, I can add you for pushing releases.

**Comment by kirs** [comment]
Thanks! Here is my rubygems account: https://rubygems.org/profiles/Kirs

**Comment by technicalpickles** [comment]
Added, thanks!

**Comment by kirs** [comment]
@technicalpickles BTW, why do we need `capistrano-tinder.gemspec` in this repo?

**Comment by technicalpickles** [comment]
Legacy :) it was named that at some point.

**Pull request merged**